### ISSUE TITLE
Support custom k8s-app label on cluster service installation 

### ISSUE THREAD

**Issue by nettoclaudio** [coredns/helm#19]
Hi there! 👋

TL;DR:
I would like to override the `k8s-app` label to `kube-dns` rather than `coredns` when installing this Chart in cluster service mode (`isClusterService` enabled). However, actually there is no way to change that since it's pointing to the Chart name - as shown below.

https://github.com/coredns/helm/blob/c4e1c9936688c6257993c2d3c4472b2ffebf75b4/stable/coredns/templates/deployment.yaml#L10

https://github.com/coredns/helm/blob/c4e1c9936688c6257993c2d3c4472b2ffebf75b4/stable/coredns/templates/service.yaml#L24

Context:

I've used kOps to configure and launch my Kubernetes clusters so far. When provisioning the cluster, kOps determines the cluster DNS IP address (e.g. based on pod's overlay network) and handover it to nodes (over the `--cluster-dns` parameter on kubelet) and it creates a `kube-dns` Service in the cluster with a fixed `clusterIP` address as well.

Now, I would like to use the official CoreDNS Helm Chart and replace the KubeDNS one. However, I want to do this by preserving the old `kube-dns` Service to be able to avoid any outage on apps (regarding DNS resolutions, for sure 🙈). Since the selector is an immutable field in the Service spec, I can't change it to accord with the CoreDNS one (`k8s-app==coredns`). So, I would like to be able to override the `k8s-app` label. 😬

**Comment by Raj-Popat** [comment]
When is this going to be merged?? And what are you doing for the time being

**Comment by JMoserCricut** [comment]
You can use (customLabels)[https://github.com/coredns/helm/blob/master/charts/coredns/values.yaml#L222] unfortunately there's a minor bug that will break the pod disruption budget. Personally this helm chart was overcomplicated so I took the existing version and copied it to our local repo to fix the PDB issue I mentioned
